alibi_detect/saving/schemas.py

```diff
@@ -13,31 +13,16 @@
     For detector pydantic models, the fields match the corresponding detector's args/kwargs. Refer to the
     detector's api docs for a full description of each arg/kwarg.
 """
-
+from __future__ import annotations
```

Comment:
What was the reason for this? Could be potentially breaking?

Reply:
Ooops thanks, I was experimenting and forgot to remove this. Deffo shouldn't be here!

Reply:
Removed in dc686f35b1120ede44268106ae00be4c66bbb6df.